docs/ref/utils.txt

```diff
@@ -928,7 +928,7 @@ appropriate entities.
 
         ``override`` is now usable as a function decorator.
 
-.. function:: localtime(value, timezone=None)
+.. function:: localtime(value=None, timezone=None)
```

Comment:
This needs an explanation of what happens when `value` isn't provided and a `.. versionchanged:: 1.9` directive.